Source/WebCore/animation/CSSAnimation.cpp

```diff
@@ -136,6 +136,8 @@ void CSSAnimation::syncPropertiesWithBackingAnimation()
         );
     }
 
+    setRange(animation.range());
```

Comment:
You should either add a new overridden property type and follow it, or leave a note to had this (preferably with a matching bug) so we do it later. You can follow the pattern used elsewhere to add this overridden property which will requires adding a bindings-specific setter to `WebAnimation` to determine when the `rangeStart` and `rangeEnd` properties may have been set via the bindings.

Reply:
Made the setters binding specific and added check for if they have been set in `CSSAnimation::syncPropertiesWithBackingAnimation`.